pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -255,15 +253,12 @@ def pre_configure_ddp(self):
         # when not all parameter backward hooks are fired by the autograd engine even if require_grad is set to True.
         # This flag does come with a performance hit, so it is suggested to disable in cases where it is possible.
         self._ddp_kwargs["find_unused_parameters"] = self._ddp_kwargs.get("find_unused_parameters", True)
-        # todo: PyTorch 1.7.0 DDP introduces `self.reducer._rebuild_buckets()` breaking manual_optimization
-        if (
-            _TORCH_GREATER_EQUAL_1_7
-            and not self.lightning_module.automatic_optimization
-            and not self._ddp_kwargs.get("find_unused_parameters", False)
+        if not self.lightning_module.automatic_optimization and not self._ddp_kwargs.get(
+            "find_unused_parameters", False
         ):
+            # TODO: PyTorch 1.7.0 DDP introduces `self.reducer._rebuild_buckets()` breaking manual_optimization
             rank_zero_warn(
-                "From PyTorch 1.7.0, Lightning ``manual_optimization`` needs to set ``find_unused_parameters=True`` "
-                "to properly work with DDP."
+                "Lightning `manual_optimization` needs to set `find_unused_parameters=True` to properly work with DDP."
```

Comment:
Maybe notifying users that `find_unused_parameters` is flipped?
```suggestion
                "Lightning `manual_optimization` needs to set `find_unused_parameters=True` to properly work with DDP."
                " Using `find_unused_parameters=True`."
```